test/Transforms/loop-fusion.mlir

```diff
@@ -2266,3 +2268,76 @@ func @affine_2_dependent_mm_fused(%arg0: memref<1024x1024xf32>, %arg1: memref<10
   // CHECK-NEXT:   }
   return
 }
+
+// -----
+
+// CHECK-LABEL: func @should_fuse_self_dependence_multi_store_producer() {
+func @should_fuse_self_dependence_multi_store_producer() {
```

Comment:
Do you have any multi-store fusion unit tests where one of the fused loops stores to a live out memref?
func @test(%live_out : : memref<64x9xi32>) {
 // Add loop nest here which stores to %live_out
 // Add other loop nests to fuse with above loop nest.
 return %live_out : memref<64x9xi32>
}

Reply:
Is the one in 2325 (`should_fuse_function_live_out_multi_store_producer`) what you are looking for?

Reply:
Yes. Thanks.